test/e2e/federation-ingress.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
`federated-ingress.go`, the verb form, meaning the Ingress itself is federated, as opposed to an Ingress that is just part of a federation.